Snowflake.Data.Tests/UnitTests/ArrowResultChunkTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿/*
+ * Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System;
+using System.Linq;
+using Apache.Arrow;
+using NUnit.Framework;
+using Snowflake.Data.Core;
+
+namespace Snowflake.Data.Tests.UnitTests
+{
+    [TestFixture]
+    class ArrowResultChunkTest
+    {
+        private static int s_rowCount = 10;
```

Comment:
why static?

Reply:
I would just change to const which also is a kind of static field.